addons/payment_adyen/static/src/js/payment_form.js

```diff
@@ -173,6 +173,28 @@ odoo.define('payment_adyen.payment_form', require => {
                             showPaymentMethods: true,
                             showPayButton: false,
                             setStatusAutomatically: true,
+                            onSelect: (component) => {
+                                if (component.props.name === "PayPal") {
+                                     if (!this.paypalForm){
+                                         // create and mount native PayPal button in the component
+                                         const component = checkout.create(
+                                             "paypal"
+                                         ).mount(`#o_adyen_paypal_component`);
```

Comment:
How does AdyenCheckout know it should be a component, not the whole dropin? Because we pass 'dropin' rather than a string that matches a payment method code?